crates/rust-analyzer/src/handlers.rs

```diff
@@ -928,7 +928,10 @@ pub(crate) fn handle_formatting(
 
     if !output.status.success() {
         match output.status.code() {
-            Some(1) if !captured_stderr.contains("not installed") => {
+            Some(1)
+                if !captured_stderr.contains("not installed")
+                    && !captured_stderr.contains("not available") =>
```

Comment:
That's an excellent example for one of our style-guide rules! :-)

https://github.com/rust-analyzer/rust-analyzer/pull/8337